.travis.yml

```diff
@@ -61,6 +65,7 @@ env:
     - MYSQL_IMAGE=mariadb:10.4                                                POSTGRES_IMAGE=postgres:9.6 PMM_SERVER_IMAGE=perconalab/pmm-server:dev-latest
 
 before_install:
+  - echo "$(DOCKER_PASSWORD)" | docker login -u "$(DOCKER_USERNAME)" --password-stdin
```

Comment:
The syntax for `docker login` is different when it's called from `travis.yml` (pls refer to https://docs.travis-ci.com/user/docker/#private-registry-login)
`echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin`